packages/web-embed-api/src/interfaces.ts

```diff
@@ -27,20 +27,33 @@ export interface IFlowAnswers {
   [answerKey: string]: IAnswer;
 }
 
-interface IWebEmbedEventData<Type extends WebEmbedMessage = WebEmbedMessage> {
+export interface BaseEventProps {
```

Comment:
We use `I` prefix for interfaces in the main and oss codebase, I'd use`IBaseEventProps` and `IRedirectEventProps` for event props

Reply:
Will update to use `I` prefix